change/@internal-react-composites-43594bee-c7ad-484e-a7e6-844473bba4f6.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "addd callreadiness opt in option for call composite",
```

Comment:
"addd callreadiness opt in option for call composite",
to
"Add enable call readiness option for Call and CallWithChat composite"

Reply:
fixed